core/shared/src/main/scala/org/http4s/EntityDecoder.scala

```diff
@@ -355,6 +355,7 @@ object EntityDecoder {
   ): EntityDecoder[F, Multipart[F]] =
     MultipartDecoder.mixedMultipart(headerLimit, maxSizeBeforeWrite, maxParts, failOnLimit)
 
+  @deprecated("Broken. An entity decoder cannot return a Stream", "0.23.17")
   implicit def eventStream[F[_]: Applicative]: EntityDecoder[F, EventStream[F]] =
```

Comment:
but you decided to keep it `implicit` didn't you?
I mean, IIUC, if it is completely broken, then it would be safer to un-`implicit` it and thus keep it there for the sake of bin-compat only. Perhaps, I am missing something though...

Reply:
No, it's a good point. I thought about this too. For a downstream with fatal deprecation warnings the end result is similar: compile will error out. The difference is whether the message will be "this is broken, don't use this" or "implicit not found".

In any case, I only added this in the last release, and given that it's broken I doubt it has found much adoption :)